multiqc/modules/deeptools/bamPEFragmentSize.py

```diff
@@ -0,0 +1,173 @@
+            headersSE["Read Length 1st Quartile"] = {'title': 'Read Len. 1st Qu.', 'description': '1st quartile read length'}
+            headersSE["Read Length Mean"] = {'title': 'Read Len. Mean', 'description': 'Mean read length'}
+            headersSE["Read Length Median"] = {'title': 'Read Len. Median', 'description': 'Median read length'}
+            headersSE["Read Length 3rd Quartile"] = {'title': 'Read Len. 3rd Qu.', 'description': '3rd quartile read length'}
+            headersSE["Read Length Maximum"] = {'title': 'Read Len. Max.', 'description': 'Maximum read length'}
+            headersSE["Read Length Std. Dev."] = {'title': 'Read Len. Std. Dev.', 'description': 'read length standard deviation'}
+            headersSE["Read Length Med. Abs. Dev."] = {'title': 'Read Len. MAD', 'description': 'read length median absolute deviation'}
+            self.add_section(name="bamPEFragmentSize (read length metrics)",
+                             anchor="bamPEFragmentSize",
+                             plot=table.plot(self.deeptools_bamPEFragmentSize, headersSE))
+
+            headersPE = OrderedDict()
+            headersPE["Fragments Sampled"] = {'title': 'Fragments Sampled'}
+            headersPE["Fragment Length Minimum"] = {'title': 'Fragment Len. Min.', 'description': 'Minimum read length'}
+            headersPE["Fragment Length 1st Quartile"] = {'title': 'Fragment Len. 1st Qu.', 'description': '1st quartile read length'}
+            headersPE["Fragment Length Mean"] = {'title': 'Fragment Len. Mean', 'description': 'Mean read length'}
+            headersPE["Fragment Length Median"] = {'title': 'Fragment Len. Median', 'description': 'Median read length'}
+            headersPE["Fragment Length 3rd Quartile"] = {'title': 'Fragment Len. 3rd Qu.', 'description': '3rd quartile read length'}
+            headersPE["Fragment Length Maximum"] = {'title': 'Fragment Len. Max.', 'description': 'Maximum read length'}
+            headersPE["Fragment Length Std. Dev."] = {'title': 'Fragment Len. Std. Dev.', 'description': 'read length standard deviation'}
+            headersPE["Fragment Length Med. Abs. Dev."] = {'title': 'Fragment Len. MAD', 'description': 'read length median absolute deviation'}
+
+            # Are there any PE datasets?
+            PE = False
+            for k, v in self.deeptools_bamPEFragmentSize.items():
+                if 'Fragment Length Minimum' in v:
+                    PE = True
+                    break
+            if PE:
+                self.add_section(name="bamPEFragmentSize (fragment length metrics)",
+                                 anchor="bamPEFragmentSize",
+                                 plot=table.plot(self.deeptools_bamPEFragmentSize, headersPE))
+
+            # Read length plot
+            config = {'data_labels': [dict(name="Read length distribution", title="Read length distribution", ylab="Read length (bases)"),
```

Comment:
Please add an `id` (most visible when exporting the plot) and `title` to all plots.